orchagent/intfsorch.cpp

```diff
@@ -419,6 +419,10 @@ void IntfsOrch::doTask(Consumer &consumer)
             {
                 vnet_name = value;
             }
+            else if (field == "mac_addr")
+            {
+                mac = MacAddress(value);
```

Comment:
As MacAddress(value) may throw invalid_argument, it is better to put this statement around 
```
try
{
    mac = MacAddress(value);
}
catch (const std::invalid_argument &e)
{
    ...
}
```

Reply:
Good point, done